src/vivarium_gates_mncnh/constants/data_keys.py

```diff
@@ -208,9 +208,9 @@ def log_name(self):
 class __NoCPAPRisk(NamedTuple):
     # Keys that will be loaded into the artifact. must have a colon type declaration
     P_RDS: str = "intervention.no_cpap_risk.p_rds"
-    P_CPAP_BEmONC: str = "intervention.no_cpap_risk.probability_CPAP_BEMONC"
```

Comment:
All the data keys updates are just to be consistent so creating the super classes is easier.